lychee-bin/src/main.rs

```diff
@@ -122,10 +123,10 @@ fn run_main() -> Result<i32> {
             tokio::runtime::Builder::new_multi_thread()
                 .worker_threads(threads)
                 .enable_all()
-                .build()?
+                .build()
         }
-        None => tokio::runtime::Runtime::new()?,
-    };
+        None => tokio::runtime::Runtime::new(),
+    }?;
```

Comment:
ok. but easy to miss when reading.

Reply:
We are creating a runtime in both cases and are checking if that failed after which I find more readable.

Reply:
the `?` operator in `}?;` is quite easy to miss.